apps/passport-client/src/dispatch.ts

```diff
@@ -341,13 +341,29 @@ async function sync(state: ZuState, update: ZuUpdate) {
     update({
       downloadingPCDs: true,
     });
-    const pcds = await downloadStorage();
-    update({
-      downloadedPCDs: true,
-      downloadingPCDs: false,
-      pcds: pcds,
-      uploadedUploadId: state.pcds.getUploadId(),
-    });
+    try {
+      const pcds = await downloadStorage();
+      update({
+        downloadedPCDs: true,
+        downloadingPCDs: false,
+        pcds: pcds,
+        uploadedUploadId: state.pcds.getUploadId(),
```

Comment:
dont we want to get the "upload id" (concatented ids) of the *new* `pcds` list we just loaded?

vs `state.pcds`